tools/console/src/test/java/org/eclipse/rdf4j/console/command/ExportTest.java

```diff
@@ -52,8 +53,9 @@ public void setUp() throws IOException, RDF4JException {
 	public final void testExportAll() throws RepositoryException, IOException {
 		File nq = new File(locationFile, "all.nq");
 		cmd.execute("export", nq.getAbsolutePath());
-		Model exp = Rio.parse(Files.newReader(nq, StandardCharsets.UTF_8), "http://example.com", RDFFormat.NQUADS);
-
+		Reader reader = Files.newReader(nq, StandardCharsets.UTF_8);
+		Model exp = Rio.parse(reader, "http://example.com", RDFFormat.NQUADS);
+		reader.close();
```

Comment:
If the parse fails this will still result in a dangling reader. Not sure that's really a major problem, but if we're cleaning it up anyway: can we use a try-with-resources construction here instead?

Reply:
sure, will do

Reply:
done